internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -7090,6 +7090,498 @@ func TestHandleGetVoteInfo(t *testing.T) {
+			return chain
+		}(),
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInternal.Code,
+	}, {
+		name:    "handleGetRawTransaction: unable to fetch treasury agenda status",
+		handler: handleGetRawTransaction,
+		cmd: &types.GetRawTransactionCmd{
+			Txid:    txid,
+			Verbose: &verboseTx,
+		},
+		mockTxMempooler: func() *testTxMempooler {
+			mp := defaultMockTxMempooler()
+			mp.fetchTransactionErr = errors.New("unable to fetch tx from mempool")
+			return mp
+		}(),
+		mockTxIndexer: func() *testTxIndexer {
+			idx := defaultMockTxIndexer()
+			idx.entry = func(hash *chainhash.Hash) (*indexers.TxIndexEntry, error) {
+				return tx0TestTx.indexEntry, nil
+			}
+			return idx
+		}(),
+		mockDB: func() *testDB {
+			db := defaultMockDB()
+			db.viewTx = &testDatabaseTx{
+				fetchBlockRegion: func(region *database.BlockRegion) ([]byte, error) {
+					return hexToBytes(tx0TestTx.hex), nil
+				},
+			}
+			return db
+		}(),
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.treasuryActive = false
+			chain.treasuryActiveErr =
+				errors.New("unable to fetch treasury agenda status")
+			return chain
+		}(),
+		wantErr: true,
+		errCode: dcrjson.ErrRPCInternal.Code,
+	}, {
+		name:    "handleGetRawTransaction: ok, verbose",
+		handler: handleGetRawTransaction,
+		cmd: &types.GetRawTransactionCmd{
+			Txid:    txid,
+			Verbose: &verboseTx,
+		},
+		mockTxMempooler: func() *testTxMempooler {
+			mp := defaultMockTxMempooler()
+			mp.fetchTransactionErr = errors.New("unable to fetch tx from mempool")
+			return mp
+		}(),
+		mockTxIndexer: func() *testTxIndexer {
+			idx := defaultMockTxIndexer()
+			idx.entry = func(hash *chainhash.Hash) (*indexers.TxIndexEntry, error) {
+				return tx0TestTx.indexEntry, nil
+			}
+			return idx
+		}(),
+		mockDB: func() *testDB {
+			db := defaultMockDB()
+			db.viewTx = &testDatabaseTx{
+				fetchBlockRegion: func(region *database.BlockRegion) ([]byte, error) {
+					return hexToBytes(tx0TestTx.hex), nil
+				},
+			}
+			return db
+		}(),
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.treasuryActive = true
+			return chain
+		}(),
+		result: verboseResult,
+	}, {
+		name:    "handleGetRawTransaction: ok, verbose, tx from mempool",
+		handler: handleGetRawTransaction,
+		cmd: &types.GetRawTransactionCmd{
+			Txid:    txid,
+			Verbose: &verboseTx,
+		},
+		mockTxMempooler: func() *testTxMempooler {
+			mp := defaultMockTxMempooler()
+			mp.fetchTransaction = dcrutil.NewTx(&tx)
+			mp.fetchTransactionErr = nil
+			return mp
+		}(),
+		mockDB: func() *testDB {
+			db := defaultMockDB()
+			db.viewTx = &testDatabaseTx{
+				fetchBlockRegion: func(region *database.BlockRegion) ([]byte, error) {
+					return hexToBytes(tx0TestTx.hex), nil
+				},
+			}
+			return db
+		}(),
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.treasuryActive = true
+			return chain
+		}(),
```

Comment:
`treasuryActive` is set to `true` by default in `defaultMockRPCChain`, so you shouldn't need to specify this here and below.